crates/uv-trampoline/src/bounce.rs

```diff
@@ -47,8 +47,8 @@ enum TrampolineKind {
 impl TrampolineKind {
     const fn magic_number(&self) -> &'static [u8; 4] {
         match self {
-            Self::Script => &[b'U', b'V', b'S', b'C'],
-            Self::Python => &[b'U', b'V', b'P', b'Y'],
+            Self::Script => b"UVSC",
+            Self::Python => b"UVPY",
```

Comment:
Ah whoops, you already did this. :P

Reply:
Sorry; this is one of those things that wasn't worth moving in the rebase since it was a mess